packages/driver/cypress/e2e/e2e/origin/commands/storage.cy.ts

```diff
@@ -0,0 +1,120 @@
+import { findCrossOriginLogs } from '../../../../support/utils'
+
+context('cy.origin storage', { browser: '!webkit' }, () => {
```

Comment:
With Origin going GA soon do we need a distinct `origin/commands` spec for this or can we include origin-specific tests in commands/storage moving forward

Reply:
Yeah, once the flag is gone, I think we should integrate the cy.origin() tests alongside the other related tests, especially where commands are concerned.